src/renderer/vt/XtermEngine.cpp

```diff
@@ -534,10 +534,16 @@ CATCH_RETURN();
     // decided to pass through would have gotten buffered here until the next
     // actual frame is triggered.
     //
-    // To fix this, flush here, so this string is sent to the connected terminal
-    // application.
+    // Originally, we would _Flush()  here, so this string (and anything buffered) is sent to the connected
+    // terminal application. As of GH#13710, we won't. We'll just buffer the string.

```

Comment:
nit: same comment as on another PR - signposts

Reply:
I guess what I mean is, "make the comment explain why the code is doing what it's doing, rather than explaining why we added and removed the code ourselves"